### PR TITLE
feat: ecosystem dashboard

### PR DESCRIPTION
**Pull request by sanjay-k1910**
## What?
- Implemented ecosystem dashboard
- Integrated "GetEcosystem" API
- Handled UI for ecosystem data

### CODE REVIEW

File: src/common/enums.ts [hunk 1]
```diff
@@ -36,3 +36,8 @@ export enum IssueCredentialUserText {
 	done = 'Accepted',
 	abandoned = 'Declined'
 }
+
+export enum EcosystemRoles {
```
Comment (nishad-ayanworks) [inline src/common/enums.ts:40]: Please be specific like `Ecosystem Member` or `Ecosystem Lead`

File: src/components/Ecosystem/Dashboard.tsx [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+    }, [])
+
+    return (
+        <div className="px-4 pt-6">
+            <div className="mb-4 col-span-full xl:mb-2">
+                <BreadCrumbs />
+            </div>
+            {
+                (success || failure) &&
+                <Alert
```
Comment (nishad-ayanworks) [inline src/components/Ecosystem/Dashboard.tsx:64]: Please use AlertComponent

File: src/common/enums.ts (hunk 1 above)
Comment (sanjay-k1910) [reply src/common/enums.ts:40]: Resolved.

File: src/components/Ecosystem/Dashboard.tsx (hunk 2 above)
Comment (sanjay-k1910) [reply src/components/Ecosystem/Dashboard.tsx]: Resolved.

**Review by nishad-ayanworks — APPROVED** [review]
LG

**Pull request merged**